README.md

```diff
@@ -14,6 +14,14 @@ You can find a detailed guide [here](./CONTRIBUTING.md)!
 
 ### There are only 2 things in computer science: cache invalidation & naming things  — *********Phil Karlton*********
 
+## Imports Order 
+- External Libraries 
+- Internal Libraries
+- Components 
+- Types 
+- Images 
+- CSS 
+
```

Comment:
why is this changes?

Reply:
IG this is showing up from someone else's commit as it was not in my initial PR

Reply:
still this should not be there right?